.solhint.json

```diff
@@ -7,9 +7,9 @@
     "max-states-count": ["error", 18],
     "compiler-fixed": true,
     "no-simple-event-func-name": true,
-    "not-rely-on-time": false,
+    "not-rely-on-time": "off",
```

Comment:
Question, does `not-rely-on-time` lint mean you can't use `now`? If so does it make more sense to just disable it with `/* solhint-disable not-rely-on-time */` or however you do it where we use `now`?

Reply:
Yes. I suppose. Good idea I will do this!

Reply:
Only problem is, this will be A LOT of inline disables :


<img width="652" alt="Screenshot 2019-12-19 at 4 45 24 PM" src="https://user-images.githubusercontent.com/11778116/71187267-077c5080-227f-11ea-9e24-8858f9656c4e.png">


At this point, I would say we essentially want to turn it off.

Reply:
I don't have a strong opinion, but it puts more lines into the smart contract on top without any real benefit.

Reply:
I would propose either of the following two options:


1. Leave it as is with "off" for now... although this is equivalent to not including it anywhere?

2. I could disable-entire-file (once) at the top of each of the SNAPP contracts, while only disabling per use in the (more relevant) StableX contracts. 


I think option 2 would probably be the better option. Also the number of usages in the SNAPP contracts suggests that we could reduce the number by introducing some helper functions. 

What do you guys think? @josojo and @nlordell ?

Reply:
In fact my proposal number two can be implements with only 5 lines of new code.

Reply:
Lets just globally disable it as you had before. Sorry for the inconvenience, I didn't expect it to be in so many places.

Reply:
If you like option 2, then go with that, I also think its better.

Reply:
DUDE! I already reset hard and force pushed!

Reply:
I'm good either way. I hadn't seen the messages after the screenshot because of Github things.